corehq/apps/commtrack/views.py

```diff
@@ -180,7 +180,7 @@ def loc_to_payload(loc):
 
     else:
         LIMIT = 100
-        loc_types = [loc_type.name for loc_type in Domain.get_by_name(domain).location_types if not loc_type.administrative]
+        loc_types = [loc_type.name for loc_type in Domain.get_by_name(domain).location_types]
```

Comment:
since we can now assign users to administrative locations we need to stop filtering the list.